.github/workflows/label_checker.yml

```diff
@@ -12,11 +12,19 @@ on:
 
 jobs:
   check_labels:
-    if: github.event_name == 'pull_request'
+    if: contains(github.event.pull_request.labels.*.name, 'ready to merge')
     name: Check labels
     runs-on: ubuntu-latest
     steps:
-      - uses: docker://agilepathway/pull-request-label-checker:latest
+      - name: Check labels
+        uses: docker://agilepathway/pull-request-label-checker:latest
         with:
-          one_of: bugfix,feature,documentation,performance,enhancement,maintenance
+          any_of: bugfix,feature,documentation,performance,enhancement,maintenance
```

Comment:
@jni - you should like this change as it allows for more than one labels.

Reply:
I do! 😍

Reply:
Not sure, but we might need to follow up on napari/docs to ensure multiple labels are treated properly when generating sections in release notes (i.e: the thing appears multiple times)

Reply:
not in napari/docs, but if double appearance is a problem then it needs to be fixed.

Reply:
I think double appearance is what we want right?